apps/ejabberd/priv/cassandra.cql

```diff
@@ -1,4 +1,77 @@
-DROP KEYSPACE IF EXISTS mongooseim;
 CREATE KEYSPACE mongooseim
          WITH REPLICATION = {'class' : 'SimpleStrategy', 'replication_factor': 3};
 USE mongooseim;
+
+CREATE TABLE mam_message(
```

Comment:
I see. Is it possible to write a migration script from the old schema to the new one? I agree the new one is better. Also I want existing users of the old backend to upgrade MongooseIM to the newest version. This will only happen if we make the migration as easy as possible for them.